01_fib/ovlords/main.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func fib(n int) {
+
+	if n > 0 {
+
```

Comment:
This is a bit over-spaced vertically. Since the bulk of the implementation is about four lines, it is not worth spacing that out any extra. Save vertical spacing for separating large blocks along functionality lines. You'll find most Go code is written that way.